### ISSUE TITLE
[Bug] Fluid Container Model is translucent since 41.0.64

### ISSUE THREAD

**Issue by cech12** [MinecraftForge/MinecraftForge#8871]
**Minecraft Version:** 1.19

**Forge Version:** 41.0.64 (and later) (tested up to 41.0.98)

**Steps to Reproduce:**
 1. setup an item that uses the `forge:bucket` aka. `forge:fluid_container` model with `"fluid":"minecraft:water"` (or any other fluid that has a translucent texture)
 2. start the game
 3. get the item into your hotbar
 4. select it to hold it in your hand
 5. see through the bucket^^

**Description of issue:**
Since the breaking change of Forge 41.0.64 the `forge:bucket` aka. `forge:fluid_container` model generates translucent item models.

Using this model for an item.
```
{
  "parent": "forge:item/bucket",
  "loader": "forge:bucket",
  "fluid": "minecraft:water"
}
```

Since Forge 41.0.64 the item looks like that:
![bucket_64+](https://user-images.githubusercontent.com/16005054/179074543-cbbaf163-34a5-483d-8a75-95b3127e3ecd.png)

In Forge 41.0.63 the item looks like expected:
![bucket_63](https://user-images.githubusercontent.com/16005054/179074518-0d29dae9-7249-4569-bac9-8a16285867d1.png)

Here is an example mod to show the issue: https://github.com/cech12/BucketTestMod (two different branches for 41.0.63 and 41.0.64)

It looks like an old issue, that was fixed in 1.16 with this commit: https://github.com/MinecraftForge/MinecraftForge/commit/ce3d8b40cf37924caf1708cdde6842ae6fdcee31

**Comment by pupnewfster** [comment]
For reference just for potentially easier tracking of how this was fixed before, there was an initial fix done in #6747 relating to this comment https://github.com/MinecraftForge/MinecraftForge/pull/6747#issuecomment-640223904 and I believe the version that actually got merged into 1.16 is this set of changes: https://github.com/MinecraftForge/MinecraftForge/commit/ce3d8b40cf37924caf1708cdde6842ae6fdcee31

**Comment by marchermans** [comment]
We should just change the background texture since that should solve it.
Yes water is see through so that part should as well imho
